packages/lib-classifier/src/helpers/roundCoordinates/README.md

```diff
@@ -0,0 +1,42 @@
+# roundCoordinates
+
+A helper function to round a set of { x, y } coordinates to two decimal places.
```

Comment:
Docccuuumeeenntaaattiiioooonnnn! 👍

Reply:
Not needed for this PR, but I think this could be used as the basis of a custom type for rounded numbers.